src/Game/Game.css

```
@@ -0,0 +1,127 @@
+
+* {
+    font-family: 'Orbitron', sans-serif;
+}
+
+
+.app-container {
+    width: 100%;
+    height: 100vh;
+    background-color: white;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    
+  }
+  
+.board {
+    width: 50%;
+    height: 50%;
+    border-radius: 10px;
+    background-color: #909090;
+    box-shadow: 0 3px 6px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23);
+}
+
+.title-principal {
+    display: flex;
+    justify-content: center;
+    font-weight: 700;
+    font-size: 30px;
+    color: #333;
+    margin-top: 20px;
+}
+.title-home-visitors{
+    color: #333;
+    font-size: 30px;
+    font-weight: bold;
+}
+
+.score-board {
+    height: 90%;
+    display: flex;
+    justify-content: center;
+}
+
+  .left-panel {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    flex-grow: 1;
+  }
+
+  .board-points{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    font-size: 70px;
+    color: red;
+    background-color: #333;
+    width: 100px;
+    height: 100px;
+    border-radius: 10px;
+  }
+
+
+  .middle-panel {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    flex-direction: column;
+    flex-grow: 1;
+  }
+  .right-panel {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    flex-grow: 1;
+  } 
+
+  .more-less {
+      display: flex;
+      margin-top: 15px;
+  }
+
+  .button-minus {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border-radius: 10px 0px 0px 10px;
+    border: none;
+    cursor: pointer;
+    width: 40px;
+    height: 40px;
+    border: 1px solid black;
+    background-color: #333;
+    color: red;
+    font-size: 30px;
+  }
+
+  .button-plus {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    line-height: 30px;
+    border-radius: 0px 10px 10px 0px;
+    border: none;
+    cursor: pointer;
+    width: 40px;
+    height: 40px;
+    border: 1px solid black;
+    background-color: #333;
+    color: red;
+    font-size: 30px;
+}
+
+.btn-reset {
```

Do not use abbreviations in the code, they are a bad practice in general. The code is less maintainable, reusable and in most cases they are objective. Always use full words whenever you can, except for acronyms.
```suggestion
.button-reset {
```